tests/test_schemas.py

```diff
@@ -99,6 +99,10 @@ def test_anonymous_request(self):
                         url='/example/custom_list_action/',
                         action='get'
                     ),
+                    'custom_list_action': coreapi.Link(
```

Comment:
Whoops, must've been before my coffee… 😬

Perhaps the option here is to have a list for the `custom_list_action` key.  Or a dict of action-links.